packages/components/tree-v2/src/composables/useCheck.ts

```diff
@@ -0,0 +1,227 @@
+import { nextTick, ref, watch, getCurrentInstance } from 'vue'
+import {
+  NODE_CHECK_CHANGE,
+  NODE_CHECK,
+  SetOperationEnum,
+} from '../virtual-tree'
+import type { Ref } from 'vue'
+import type {
+  TreeProps,
+  TreeKey,
+  TreeNode,
+  Tree,
+  TreeNodeData,
+} from '../tree.type'
+
+export function useCheck(props: TreeProps, tree: Ref<Tree | null>) {
```

Comment:
We'd better to use `undefined` instead `null` (because the default `ref()` is `undefined`)
```suggestion
export function useCheck(props: TreeProps, tree: Ref<Tree | undefined>) {
```